src/ssl.c

```diff
@@ -5893,8 +5898,35 @@ int AddCA(WOLFSSL_CERT_MANAGER* cm, DerBuffer** pDer, int type, int verify)
     }
 #endif
 
+#ifdef WOLFSSL_DEBUG_CERTS
+    WOLFSSL_MSG_CERT("\tParsing new CA");
+#endif
     ret = ParseCert(cert, CA_TYPE, verify, cm);
+
     WOLFSSL_MSG("\tParsed new CA");
+#ifdef WOLFSSL_DEBUG_CERTS
+    #ifdef WOLFSSL_SMALL_STACK
+    if (cert == NULL) {
+        WOLFSSL_MSG_CERT(WOLFSSL_MSG_CERT_INDENT "Failed; cert is NULL");
+    }
+    else
+    #endif
+    {
+        if (ret == 0) {
```

Comment:
Just define `const char* err_msg;` here and place the `(void)err_msg;` below that at line 5926 for all cases (don't wrap in `#if defined(WOLF_NO_VARIADIC_MACROS) || defined(NO_WOLFSSL_DEBUG_CERTS)`